platform/jewel/ui/build.gradle.kts

```diff
@@ -16,6 +16,7 @@ dependencies {
     implementation(compose.components.resources)
     iconGeneration(libs.intellijPlatform.util.ui)
     iconGeneration(libs.intellijPlatform.icons)
+    testImplementation(projects.intUi.intUiStandalone)
```

Comment:
As we are adding the int-ui only for tests, we don't cause cyclic dependencies and have an easy configured theme to test.

Similar to this comment: https://discuss.gradle.org/t/gradle-support-for-cyclic-dependencies/14355/2